.github/workflows/release.yml

```diff
@@ -75,10 +65,19 @@ jobs:
       - run: npm config set '//npm.pkg.github.com/:_authToken' $GH_TOKEN
       - run: npm config set @typeform:registry https://npm.pkg.github.com/ # install jarvis from github
       - run: yarn add -W @typeform/jarvis
-      - run: git checkout HEAD -- package.json # do not save jarvis dependency to package.json (the file is committed by semantic-release to bump version)
+      - run: git checkout HEAD -- package.json # do not save jarvis dependency to package.json because it is private (the file is committed by semantic-release to bump version)
+
       - run: yarn release
         env:
           GITHUB_TOKEN: ${{ secrets.GH_TOKEN }}
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY }}
```

Comment:
nit: I would sort the list of env vars alphabetically - makes finding things in a long list easer